.travis.yml

```diff
@@ -36,8 +36,9 @@ install:
   - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION
   - conda env update -q -n test-env -f environment.yml
   - conda activate test-env
+  - pip install -e .
```

Comment:
Don't think we need the `-e` flag, but I suppose it doesn't really matter.

Reply:
OK, just dropped it to see if it would change anything.

Reply:
Alright the `-e` flag is needed for Travis to pass. Not sure why.